packages/x-charts/src/ChartsRootSurface/ChartsRootSurface.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import useForkRef from '@mui/utils/useForkRef';
+import { ChartsSurface, ChartsSurfaceProps } from '../ChartsSurface';
+import { useChartRootRef } from '../hooks';
+
+export interface ChartsRootSurfaceProps extends ChartsSurfaceProps {}
+
+/**
+ * `ChartsRootSurface` is a wrapper around the `ChartsSurface` component that provides a reference to the chart root element.
+ * This is only needed for charts that aren't wrapped in `ChartsWrapper`.
+ */
+export const ChartsRootSurface = React.forwardRef<SVGSVGElement, ChartsRootSurfaceProps>(
```

Comment:
Had to create this to set the chart root ref. An alternative would be to default to `svgRef` in the `useChartProExport` plugin, but I thought the current approach would be cleaner.

Reply:
This PR might solve your issue by introducing the wrapper to heatmap too
https://github.com/mui/mui-x/pull/17943

Reply:
Yes, it will. I can wait for that to be merged